vignettes/cookbook_preparatory_steps.Rmd

```diff
@@ -14,6 +14,233 @@ knitr::opts_chunk$set(
+
+### Companies to apply primary energy split on
+
+- optional input
+- self-prepared
+- CSV file
+
+When applying the sector split on company exposures, the user can provide a list of companies for which the sector split should be based on primary energy content. For all other companies, a simple equal weights split will be applied. For more information on the sector split, see the documentation in `vignette("sector_split")`.
+
+### Manual Sector Classification
+
+- optional input
+- self-prepared
+- CSV file
+
+In case the user cannot obtain sector classification codes of any of the classification systems featured in `r2dii.data::sector_classifications` (currently the following classification systems are featured: `r unique(r2dii.data::sector_classifications$code_system)`), the user can provide a manually created sector classification file for matching the loan book to the ABCD instead. Generally, any such manually prepared sector classification file must follow the format of `r2dii.data::sector_classifications`. It is recommended to use the built in sector classifications if possible, as mapping your own sector classification to the PACTA sectors can be complex and time consuming.
+
+## Required Software
+
+Using the `r2dii.*` suite of packages for the PACTA for Banks analysis requires the following software to be installed on your system:
+
+### R (version 4.1.0 or higher)
+
+R is the programming language that the `r2dii.*` suite of packages is written in. You can download R from the [Comprehensive R Archive Network (CRAN)](https://cran.r-project.org/).
+
+### RStudio (optional)
+
+RStudio is an integrated development environment (IDE) for R developed by Posit. It is not strictly required to run the analysis, but it can be helpful for managing your project and running the analysis. Generally, RStudio is very widely used among the R community and probably the easiest way to interact with most R tools, such as the `r2dii.*` suite of packages. RStudio Desktop is an open source tool and free of charge. You can download RStudio from the [Posit RStudio website](https://posit.co/downloads/).
+
+### `r2dii.*` suite of R packages
+
+The `r2dii.*` suite of R packages is the main software tool that you will use to run the PACTA for Banks analysis.
+
+::: {.pkgdown-release}
+You can install the `r2dii.*` suite of R packages from any CRAN mirror by running the following command in R:
+
+``` r
+install.packages(c("r2dii.data", "r2dii.match", "r2dii.analysis", "r2dii.plot"))
+```
+:::
+
+::: {.pkgdown-devel}
+You can install the development versions of the `r2dii.*` suite of R packages from GitHub with:
+
+``` r
+# install.packages("pak")
+pak::pak(c("RMI-PACTA/r2dii.data", "RMI-PACTA/r2dii.match", "RMI-PACTA/r2dii.analysis", "RMI-PACTA/r2dii.plot"))
+```
+
+We use the [`pak` package](https://pak.r-lib.org/index.html) as a simple [tool to install packages from GitHub](https://pak.r-lib.org/reference/pak.html).
+:::
+
+#### Connecting to GitHub from RStudio
```

Comment:
keeping this depends on whether or not we think there is any merit in keeping a description for downloading from github. If not, we might as well remove this section, no?